custom_components/elasticsearch/logger.py

```diff
@@ -2,4 +2,26 @@
 
 import logging
 
-LOGGER = logging.getLogger("custom_components.elasticsearch")
+LOGGER = logging.getLogger("es_integration")
```

Comment:
Does this still work when configuring logging via yml? https://www.home-assistant.io/integrations/logger/